src/views/user/welcome.ejs

```diff
@@ -1,32 +1,35 @@
-<div class="card-container">
-    <div class="fr-card">
-        <div class="fr-card__body">
-            <div class="fr-card__content">
-                <div>
-                    <img src="/dist/tada.svg" alt="">
-                </div>
-                <br>
-                <h1 class="fr-card__title">
-                    Votre compte est créé !
-                </h1>
-                <div class="fr-card__desc">
+<div class="fr-grid-row fr-grid-row--center fr-grid-row--middle mcp-full-height">
+    <div class="fr-mb-6w fr-col-lg-5 fr-col-offset-lg-1--right fr-col-md-7 fr-col-sm-8 fr-pt-5w mcp-full-height">
+        <div>
+            <div>
+                <img src="/dist/tada.svg" alt="">
+            </div>
```

Comment:
quickwin: we should be able to remove this (and delete the associated tada.svg file)